Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -1472,12 +1490,20 @@ def config_search_command(client: Client, args: Dict[str, Any]) -> CommandResult
                                      time_from=args.get('time_range_date_from'),
                                      time_to=args.get('time_range_date_to'))
     search_id = args.get('search_id')
+
     sort_direction = args.get('sort_direction', 'desc')
     sort_field = args.get('sort_field', 'insertTs')
     if any([sort_direction, sort_field]) and not all([sort_direction, sort_field]):
         raise DemistoException('Both sort direction and field must be specified if sorting.')
 
-    response = client.config_search_request(time_filter, str(query), limit, search_id, sort_direction, sort_field)
+    include_resource_json = args.get('include_resource_json', 'true')
+    include_additional_resource_fields = args.get('include_additional_resource_fields', 'false')
```

Comment:
```suggestion
    include_additional_resource_fields = argToBoolean(args.get('include_additional_resource_fields', 'false'))
```

Reply:
I use the string value as is, without the need to go through a bool.

Reply:
[Done](https://github.com/demisto/content/pull/32606/commits/5fa21493d916ad6da2ccfe22910449fdbc49d6d5).